manual/overview.wiki

```diff
@@ -84,7 +84,11 @@ functions are optimized:
   * trampolines are used otherwise.
 <<a_manual chapter="tailcall" |More about tail call optimization>>.
 
-Effect handlers are fully supported with the {{{--enable=effects}}} flag. This is not the default for now since effects are not widely used at the moment and the generated code can be slower, larger and less readable.
+Effect handlers are fully supported with the
+{{{--enable=effects}}} flag. Effect support is disabled by
```

Comment:
This needs to be updated

Reply:
Shouldn’t we wait for the new flag to be supported by Dune before documenting it? See https://github.com/ocsigen/js_of_ocaml/pull/1461#issuecomment-2545773411.

Reply:
Sounds good